### PR TITLE
Help Center: Load attachments from zendesk

### PR DESCRIPTION
**Pull request by renancarvalho**
Related to https://github.com/Automattic/dotcom-forge/issues/9644

## Proposed Changes

* Make help center chat capable to load images and links from a message sent from someone using Zendesk
* This is a first step to allow users to send attachment files to Zendesk.

image

## Why are these changes being made?

* Improve chat experience and features

## Testing Instructions

* Use live link and add the flag `?flags=help-center-experience`
* Start a new chat and ask `odie` to talk to a human
* Open Zendesk and reply yourself.
* From ZENDESK send attachments like images or texts with link
* It should load properly on HelpCenter

The other way around will be implemented in a followup PR

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

**Review by escapemanuele — APPROVED** [review]
So good to see images there, thank you for this!

File: packages/odie-client/src/data/use-get-zendesk-conversation.ts [hunk 1]
```diff
@@ -1,12 +1,10 @@
 import Smooch from 'smooch';
+import { zendeskMessageConverter } from '../utils';
+import type { ZendeskMessage, Conversation } from '../types';
```
Comment (escapemanuele) [inline packages/odie-client/src/data/use-get-zendesk-conversation.ts]:
```suggestion
import type { ZendeskMessage, Conversation } from '../types/';
```

File: packages/odie-client/src/components/message/custom-a-link.tsx [hunk 2]
```diff
@@ -41,7 +43,7 @@ const CustomALink = ( {
 			<a
 				className="odie-sources-link"
 				href={ transformedHref }
-				target="_self"
```
Comment (agrullon95) [inline packages/odie-client/src/components/message/custom-a-link.tsx:44]: We should fallback to  `_self` if `target` doesn't exist.

Comment (renancarvalho) [reply packages/odie-client/src/components/message/custom-a-link.tsx:44]: It is, the default value for it is _self, check [here](https://github.com/Automattic/wp-calypso/pull/96068/files#diff-56b82e97e9256975f119fd158e7e9a5d37824c638c1168daae01d72b820a2044R16)

**Review by agrullon95 — APPROVED** [review]
Looks awesome and works well. I tested with a text message, image, and mp4 file.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/16988239

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @renancarvalho for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16988239).

**Pull request merged**